.github/workflows/web.yml

```diff
@@ -68,11 +68,19 @@ jobs:
 
       - name: Build the collection
         run: |
+          # Pin versions to speed up CI
+          sed -i 's/ansible\.windows:\s*"\*"/ansible.windows: "2.3.0"/' galaxy.yml
```

Comment:
Why do this?  In my mind we should be testing against what we publish in the galaxy.yml file otherwise we may have an issue of experiencing/not experiencing failures that we should/shouldn't during CI/CD testing.  Not to mention having to change versions in 5 places instead one 1.

Reply:
This part where galaxy looks up every single version, for every single dependency where we use `*` takes multiple minutes, multiply that by the matrix of configurations we're testing in github workflows, you're looking at a couple hours just to resolve these dependencies. (Yes, it's really that stupid, I've provided links to reported issues in ce6b2dae76708a55803081523c87f3080d70f56c)

I think it's better we pin this for CI only, as a user could have installed a different collection that pins postgresql to X.Y.Z and if we're fine with that (given we set it to `*`), galaxy will say requirements are already met and leave it alone anyway, right?

Reply:
Pinning in CI is not ideal, and especially since it's in this many places, but pinning in galaxy.yml would be the worse of two options.

In any case we should probably specify minimum requirements for all the `*` dependencies though.